doc/swww-img.1.scd

```diff
@@ -218,8 +218,7 @@ The images sent will be cached at _$XDG_CACHE_HOME/swww_ or _$HOME/.cache/swww_
 if $XDG_CACHE_HOME does not exist. For each monitor, there will be a file in
 those locations corresponding to the current image/animation being displayed.
 Importantly, **cache will only be loaded during initialization if you use swww
-init**. That is, calling `swww-daemon` directly will **NOT** load the cache, but
-calling `swww-init` will.
+init**. That is, calling `swww-daemon` directly will **NOT** load the cache.
```

Comment:
Ah, this isn't true anymore. `swww-daemon` loads the cache now, unless you pass it the `--no-cache`  flag.